src/components/LogWindow.jsx

```diff
@@ -126,10 +152,38 @@ export default class LogWindow extends Component {
         <Card.Header>{this.props.title}</Card.Header>
         <Card.Text>
           <Container className={'log-container'}>
+            <Form>
+              <InputGroup size={'sm'}>
+                <InputGroup.Text>
+                  <i className="bi bi-filter"></i>
+                </InputGroup.Text>
+                <InputGroup.Prepend>
+                  <InputGroup.Text>Status:</InputGroup.Text>
+                </InputGroup.Prepend>
+                <Select
+                  value={this.state.filterStatusOption}
+                  onChange={this.handleStatusFilterChange}
+                  options={this.statusOptions}
+                  defaultValue={this.statusOptions[0]}
+                  menuPortalTarget={document.querySelector('body')}
+                  styles={{
+                    control: (baseStyles, state) => ({
+                      ...baseStyles,
+                      width: '12rem'
+                    }),
+                  }}
+                />
+              </InputGroup>
+            </Form>
             <Card className={'log-card'}>
               <Card.Text>
                 <Container className={'log-container'}>
-                  {this.state.items.map(item => {
+                  {this.state.items.filter((item) => {
+                    if (this.state.filterStatusOption && this.state.filterStatusOption.value !== 'any') {
+                      return item.status === this.state.filterStatusOption.value;
+                    }
+                    return true;
```

Comment:
Micro optimization nit, but filter status is value & not "any" before even filtering to avoid unnecessarily checking for every value

```javascript
((this.state.filterStatusOptions && this.state.filterStatusOption.value !== 'any')
    ? this.state.items.filter(item => item.status === this.state.filterStatusOptions.value)
    : this.state.items)
.map(item => {
// ...

```

Reply:
good point

Reply:
fixed